plugins/outputs/health/health.go

```diff
@@ -219,10 +231,12 @@ func (h *Health) isHealthy() bool {
 
 func NewHealth() *Health {
 	return &Health{
-		ServiceAddress: defaultServiceAddress,
-		ReadTimeout:    config.Duration(defaultReadTimeout),
-		WriteTimeout:   config.Duration(defaultWriteTimeout),
-		healthy:        true,
+		ServiceAddress:               defaultServiceAddress,
+		ReadTimeout:                  config.Duration(defaultReadTimeout),
+		WriteTimeout:                 config.Duration(defaultWriteTimeout),
+		UnhealthyIfMetricsAreDelayed: false,
+		healthy:                      true,
+		firstMetricReceived:          false,
```

Comment:
This change is unnecessary if we constrain the check to a positive value (i.e. > 0) for the time check.

Reply:
removed both boolean flags `UnhealthyIfMetricsAreDelayed` and `firstMetricReceived`. Used only the max delay and the last metric time.